BadHabbit/BadHabbit/UserStories/SignInScreen/SignInViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  SignInViewController.swift
+//  BadHabbit
+//
+//  Created by Halina Kurylchykava on 17.06.23.
+//
+
+import UIKit
+import AuthenticationServices
+
+class SignInViewController: UIViewController {
+    let userIdentifierLabel = UILabel()
```

Comment:
Don't forget about marking private what shouldn't be used outside of this class

Reply:
changed